var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -265,7 +268,7 @@ def flag_handler(self, name, flags):
 
     def setup_build_environment(self, env):
         spec = self.spec
-        if '+ownlibs' in spec:
+        if '+ownlibs' in spec and 'platform=windows' in spec:
```

Comment:
Is this intentional: I thought `CMake` wouldn't depend on `openssl` when `platform=windows` so I would assume this should contain a `not`.

It also occurs to me we should add `conflicts(~ownlibs, when="platform=windows")` since I'm assuming this also shouldn't build its own `curl` (for the same reason Spack shouldn't build a separate `openssl` package on Windows).

Reply:
The first issue is a typo.

The second, I don't know that we want to go as far as adding the conflict, that may be something to discuss at the meeting tomorrow. It may be that we just want to extend to curl the option to build with the Windows SSL library or something similar.